### PR TITLE
Add frontend build instructions to quickstart section of README

### PR DESCRIPTION
**Pull request by guergana**

### CODE REVIEW

**Comment by lucaswerkmeister** [comment]
I’m not sure we need to list `npm run watch` here – maybe it’s enough to say `npm run dev`, and then add a kind of “see also” link pointing to the “frontend – working with CSS and JS” section?

**Comment by micgro42** [comment]
Given that the Quickstart includes installation instructions for composer dependencies, it should probably also contain the installation instructions for the node dependencies.

**Comment by guergana** [comment]
> 

You are right. i wasn't aware that they are not already there. will add

**Review by micgro42 — COMMENTED** [review]
We should probably not be ambiguous in a Quickstart section of the documentation and only give a single command.

I recommend the `npm run dev` one, as that adds the least overhead (both mental and in terms of system load).

For the day-to-day development instructions, one can have a look at the rest of the README.md

**Comment by micgro42** [comment]
> We should probably not be ambiguous in a Quickstart section of the documentation and only give a single command.
> 
> I recommend the `npm run dev` one, as that adds the least overhead (both mental and in terms of system load).
> 
> For the day-to-day development instructions, one can have a look at the rest of the README.md

I didn't see Lucas already wrote something to that effect, sorry

File: composer.lock [hunk 1]
```diff
@@ -124,28 +124,28 @@
         },
         {
             "name": "doctrine/inflector",
-            "version": "2.0.4",
```
Comment (micgro42) [inline composer.lock:127]: This lock file was probably added by accident to the commit and should be removed. Updating dependencies should probably happen in a dedicated commit and PR.

Comment (guergana) [reply composer.lock:127]: oh, yes, weird. i dont recall making any composer changes

Comment (guergana) [reply composer.lock]: it's fixed :)

**Review by micgro42 — APPROVED** [review]

**Pull request merged**